bin/ycsb

```diff
@@ -107,9 +108,15 @@ command = COMMANDS[sys.argv[1]]["command"]
 database = sys.argv[2]
 db_classname = DATABASES[database]
 options = sys.argv[3:]
+java_home = os.environ["JAVA_HOME"]
```

Comment:
the java home stuff is good, but seems unrelated to the rest of the mongo improvements. If I file an issue for "allow the use of java home" could you edit the commit message to call out closing it?